### DISCUSSION TITLE
Remove private tags

### DISCUSSION THREAD

**Discussion by drewen1** [fo-dicom/fo-dicom#1685] [Q&A] [answered]
Is there an API to remove all private tags? Thank you.

**Comment by mrbean-bremen** [comment]


**Reply by drewen1** [reply]
Thank you for this answer, I did the test its workable.

**Comment by mrbean-bremen** [comment, marked as answer]
There is no special API, but you should be able to do this easily by filtering the private tags, something like:
```cs
dataset.Remove(x => x.Tag.IsPrivate);
```
